usecases/linearFunction/test_optimize.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+import numpy as np
+from scipy.optimize import least_squares
+from pathlib import Path
+import yaml
+from linear_model_error import LinearModelError
+from bayes.multi_model_error import MultiModelError
+
+
+class MultiLinearModelError(MultiModelError):
```

Comment:
The idea to define that as a class is to provide (in the future) functions that generate the relevant data structures from database queries etc.. So instead of providing yaml files, there will be add_model_error methods that might are from different interfaces. Since there is currently only the yaml files, this is directly added to the constructor.